rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -40,3 +51,50 @@ u64 _sys_game_get_system_sw_version()
 {
 	return stof(utils::get_firmware_version()) * 10000;
 }
+
+error_code _sys_game_watchdog_start(u32 time)
```

Comment:
what if this is called multiple times?
you'll spawn thousands of threads...

Reply:
> what if this is called multiple times? you'll spawn thousands of threads...

In the log, I can see that the game just called this function once when booting.
It should be the game's responsibility to make sure this function is used properly.

Reply:
so you want the emulator to restart 1000 times at once if the game decides to be weird?

I'd just add a bool that checks if a watchdog exists.
Or use an fxo object that handles this.